alembic/versions/347ba3ac054f_update_image_attrs.py

```diff
@@ -22,7 +26,10 @@
 
 
 def upgrade():
-    imgs = StoredFile.query.filter_by(size=None)
+    connection = op.get_bind()
+    Session = sessionmaker(bind=connection.engine)
+    session = Session(bind=connection)
+    imgs = session.query(StoredFile).filter_by(size=None).options(load_only("id", "name", "title"))
```

Comment:
@jace that's exactly why I did this. The way it was before, the model was imported and used which was breaking the migrations. The way above, it uses the session of the migration, so it uses the version of the model at that point in migration and not the current version of the model.